src/Common/src/Interop/Windows/Winsock/Interop.GetAddrInfoExW.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Net.Sockets;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+internal static partial class Interop
+{
+    internal static partial class Winsock
+    {
+        internal unsafe delegate void GetAddrInfoExCompletionCallback([In] int error, [In] int bytes, [In] NativeOverlapped* overlapped);
+
+        [DllImport(Interop.Libraries.Ws2_32, ExactSpelling = true, CharSet = CharSet.Unicode, BestFitMapping = false, ThrowOnUnmappableChar = true, SetLastError = true)]
```

Comment:
Why the `ThrowOnUnmappableChar = true`?  That's pretty rare to see.

Reply:
I copied theses properties from the getaddrinfo declaration without thinking. It doesn't make sense since the call is Unicode, I will remove ThrowOnUnmappableChar and BestFitMapping properties.